ci/packages/package.go

```diff
@@ -92,6 +104,26 @@ func PackageLinuxDebianArm() error {
 	return env.IfRelease(storage.UploadArtifacts)
 }
 
+// PackageLinuxDebianArm64 builds and stores debian armv6l package
+func PackageLinuxDebianArmv6l() error {
+	logconfig.Bootstrap()
+	if err := goGet("github.com/debber/debber-v0.3/cmd/debber"); err != nil {
+		return err
+	}
+	envi := map[string]string{
+		"GOOS":   "linux",
+		"GOARCH": "arm",
+		"GOARM":  "6",
+	}
+	if err := sh.RunWith(envi, "bin/build"); err != nil {
+		return err
+	}
+	if err := packageDebian("build/myst/myst", "armhf"); err != nil {
```

Comment:
It looks like we just need to change the string here and we are ready to go.
```suggestion
	if err := packageDebian("build/myst/myst", "arm61"); err != nil {
```